pkg/asset/installconfig/installconfig.go

```diff
@@ -74,9 +71,9 @@ func (a *InstallConfig) Generate(parents asset.Parents) error {
 		},
 		ClusterID: clusterID.ClusterID,
 		Admin: types.Admin{
-			Email:    emailAddress.EmailAddress,
-			Password: password.Password,
-			SSHKey:   sshPublicKey.Key,
+			Password:     password.Password,
```

Comment:
From what I can tell, the entire install config in stored in the `kube-system/cluster-config-v1` config map.  We do not want the password or the hash stored there.

Reply:
ok,.. back to original plan of generating the pw in `pkg/asset/manifests/tectonic.go`, will push an update in a bit